dotnet/src/Connectors/Connectors.OpenAI/OpenAIPromptExecutionSettings.cs

```diff
@@ -174,19 +174,13 @@ public int ResultsPerPrompt
     /// Defaults to "Assistant is a large language model."
     /// </summary>
     [JsonPropertyName("chat_system_prompt")]
-    public string ChatSystemPrompt
+    public string? ChatSystemPrompt
```

Comment:
This change makes sense. Just calling out that it could introduce new warnings into someone's project if they're reading this property.

Reply:
Yes, expecting the API check to complain about this.

Reply:
I don't actually know if the API compat tooling understands nullable reference type annotations; it may not. Such changes aren't binary breaking.